theseus/embodied/collision/tests/test_eff_obj_contact.py

```diff
@@ -8,6 +8,7 @@
 import torch
 
 import theseus as th
+from theseus.core import Variable
```

Comment:
Don't need this since it is already available as `th.Variable`

Reply:
@mhmukadam Do i need to change this ?